pages/common/vue-build.md

```diff
@@ -0,0 +1,8 @@
+# vue build
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do quick prototyping.
```

Comment:
Please also remove "you to do" as I mentioned in my last comment. We prefer the tone in our pages to be in the 3rd person.

Reply:
Agreed. In which case, "enables" could perhaps work better than "allows".

Reply:
Changes applied. Thanks for your suggestions!